cppcache/src/ThinClientPoolDM.cpp

```diff
@@ -178,7 +178,23 @@ ThinClientPoolDM::ThinClientPoolDM(const char* name,
   ACE_TCHAR hostName[256];
   ACE_OS::hostname(hostName, sizeof(hostName) - 1);
   ACE_INET_Addr driver(hostName);
-  uint32_t hostAddr = driver.get_ip_address();
+
+  uint8_t hostaddr[16];
```

Comment:
Can we create a common method so we don't have duplicate IPv4 and IPv6 code in multiple places?

Reply:
I've been wondering about this all morning.  Given that there are multiple instances of this code, and they all just generate this address buffer, pass it into the ClientProxyMembershipID ctor and then forget about it, will we lose any generality if we just remove the hostaddr parameter from that ctor and do this work inside ClientProxyMembershipID?

Reply:
It's an option to try. Both are better than duplicated logic.